request/send_mam_req.h

```diff
@@ -0,0 +1,25 @@
+#ifndef REQUEST_SEND_MAM_REQ_H_
+#define REQUEST_SEND_MAM_REQ_H_
+
+#include <stdint.h>
+#include <stdlib.h>
+#include <string.h>
+#include "accelerator/errors.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+typedef struct send_mam_req_s {
+  char* payload;
+} send_mam_req_t;
```

Comment:
Maybe we can add payload size, so it's able to check size and security here.

Reply:
I am thinking should we convert the payload from `char*` to `tryte` here too